templates/index.html

```diff
@@ -12,90 +12,145 @@
+  <h1 class="mb3">Follow Us</h1>
+  <p class="lh-copy mt1 mb4">Urbit is actively being built, explored, and experimented with.
+    To follow our progress you can:
+  </p>
+  <ul class="list">
+    <li class="mt2">
+      Follow us on <a class="b--black bb" href="https://twitter.com/urbit">Twitter</a>
+    </li>
+    <li class="mt2">
+      Explore the code on
+      <a class="b--black bb" href="https://github.com/urbit/urbit">GitHub</a>
+    </li>
+    <li class="mt2">
+      Ask questions in our <a class="b--black bb" href="https://discord.gg/bkEX6EU">Discord</a>
+    </li>
+  </ul>
+  <p class="lh-copy">
+    or boot Urbit and join <code class="dt bg-gray4">~dopzod/urbit-help</code>
+  </p>
+</section>
 
-  <aside class="half-2-md half-2-lg full mt0-l mt0-xl mt4" id="newsletter-subscribe-top">
-    <p class="lh-copy mt1 mb3">We send regular updates about our project. Enter your email to get them.</p>
-    <iframe name="nothing" style="display:none;"></iframe>
-    <form action="https://urbit.us11.list-manage.com/subscribe/post?u=972a03db9e0c6c25bb58de8c8&amp;amp;id=be143888d2"
-      method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate form"
-      target="_blank" novalidate>
-      <div class="input-group" id="mc_embed_signup_scroll">
-        <div class="mc-field-group w-100 relative">
-          <input class="bg-white black b--black ba pa3 w-100 mb2 br0" type="email" name="EMAIL" id="mce-EMAIL"
-            placeholder="your@email.com">
-            <button id="mc-embedded-subscribe" class="dib bn absolute"
-          style="font-family: 'Inter UI', san-serif; right: 3px; top: 15px; -webkit-appearance: none; background: transparent;" type="submit"
-          name="subscribe" onclick="_paq.push(['trackEvent', 'Mailing List', 'Subscribe'])"><span class="fr pr1">-></span>
+<section
+  class="full c7-12-md c7-11-lg mb6 mt6-xl mt6-l mt6-m"
+  id="newsletter-subscribe-top">
+  <h1 class="mb3">Stay Up to Date</h1>
+  <p class="lh-copy mt1 mb3">
+    We send regular updates about our project. Enter your email to get them.
+  </p>
+  <iframe name="nothing" style="display:none;"></iframe>
+  <form
+    action="https://urbit.us11.list-manage.com/subscribe/post?u=972a03db9e0c6c25bb58de8c8&amp;amp;id=be143888d2"
+    method="post"
+    id="mc-embedded-subscribe-form"
+    name="mc-embedded-subscribe-form"
+    class="validate form"
+    target="_blank"
+    novalidate>
+    <div class="input-group" id="mc_embed_signup_scroll">
+      <div class="mc-field-group w-100 relative">
+        <input
+          class="bg-white black b--black ba pa3 w-100 mb2 br0"
+          type="email"
+          name="EMAIL"
+          id="mce-EMAIL"
+          placeholder="your@email.com"/>
+        <button
+          id="mc-embedded-subscribe"
+          class="dib bn absolute bg-transparent"
+          style="font-family: 'Inter UI', san-serif; right: 3px; top: 15px; -webkit-appearance: none;"
```

Comment:
missing `background: transparent`

Reply:
What is missing? I added bg-transparent which should do the trick. Could have missed something though.

Reply:
oh i see it lol ok this is smart